ArchiSteamFarm/Bot.cs

```diff
@@ -2934,93 +2957,110 @@ public sealed class Bot : IAsyncDisposable {
 		}
 
 		private async Task SendCompletedSets() {
-			ISet<uint> appIDs = await GetPossiblyCompletedBadgeAppIDs().ConfigureAwait(false);
+			await SendCompleteSetsSemaphore.WaitAsync().ConfigureAwait(false);
```

Comment:
This is easy mode, we want to ensure that we don't miss a possible set just because we've received event later on, check my approach with `OnNewTrade()` for idea.